.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Android CI
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+
+      - name: Set up JDK 17
```

Comment:
The build version is already set to `21`, and I am about to switch to `25`.

https://github.com/cygnusx-1-org/continuum/blob/61993265167165c25a89e070bcc7c75fee77393d/app/build.gradle#L131